traffic_monitor/towrap/towrap.go

```diff
@@ -431,23 +439,23 @@ func (s TrafficOpsSessionThreadsafe) CRConfigRaw(cdn string) ([]byte, error) {
 	}
 	defer s.crConfigHist.Add(hist)
 
-	crc := &tc.CRConfig{}
-	json := jsoniter.ConfigFastest
-	if err = json.Unmarshal(data, crc); err != nil {
-		err = errors.New("invalid JSON: " + err.Error())
-		hist.Err = err
-		return data, err
+	if crConfig != nil {
+		if err = json.Unmarshal(configBytes, *crConfig); err != nil {
```

Comment:
Wait, do we even need to dereference `crConfig` here? Isn't the 2nd arg supposed to be a pointer? Should `crConfig` be initialized on L381 so that it's non-nil in either case?